widgets/styles/partials/_swiper.scss

```diff
@@ -0,0 +1,13 @@
+@forward "pkg:swiper/swiper-bundle.css";
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap');
+
+:host {
+    font-family: Inter, sans-serif;
+}
+
+@font-face {
```

Comment:
We do have font.scss partial already. Let's not spread fonts everywhere.

Reply:
Fixed.